components/Templates/Subjects.js

```diff
@@ -1,40 +1,37 @@
-import {map} from 'lodash';
 import PropTypes from 'prop-types';
 import React from 'react';
 
 import Block from '../Block';
 import Button from '../Button';
-import Teaser from '../Teasers';
 import {SectionTitle} from '../Typography';
 
 const Subjects = ({
   content,
   domain,
-  lazy,
-  status,
-  theme,
-  customTitle,
   customLeftTitle,
   customCenterTitle,
   customRightTitle,
   leftActionButtonProps,
   centerActionButtonProps,
   rightActionButtonProps,
-  leftTeaserProps,
-  centerTeaserProps,
-  rightTeaserProps,
+  RenderSlot,
```

Comment:
O que é RenderSlot?

Reply:
O RenderSlot é uma function do front que alterna a renderização do componente conforme o input-template dele no Polopoly (ads.it, blocks.it, element.it, teaser.it). Parece uma responsabilidade do front/polopoly

Reply:
Me pergunto o que será necessário para essa alteração no front do costa-norte. Ao meu ponto de vista é um impacto grande pois teríamos que modificar toda a estrutura de blocos do backend. Acho que talvez a saída para isso no design-system não seja trocar o teaser pelo render slot e sim possibilitar o renderSlot em detrimento do Teaser com uma condição.